models/model_adapter.py

```diff
@@ -86,9 +95,29 @@ def load(self, local_path, **kwargs):
         if os.path.isdir(self.images_path):
             shutil.rmtree(self.images_path)
 
+    def _parse_lpr_results(self, predict_status):
```

Comment:
If you need this function move it to the LPR adapter. 
only use the main adapter predict if possible with LPR format

Reply:
Done